pipeline/inputs/statsd.md

```diff
@@ -16,6 +16,12 @@ The plugin supports the following configuration parameters:
 | Listen | Listener network interface. | 0.0.0.0 |
 | Port | UDP port where listening for connections | 8125 |
 | Threaded | Indicates whether to run this input in its own [thread](../../administration/multithreading.md#inputs). | `false` |
+| Metrics | Ingest as metrics type of events. | `off` |
+
+Note: When enabling `Metrics On`, the DogStatsD protocol of metrics can be handled.
+Including key-value format of tags:
```

Comment:
Can we link out to the docs for the definition?

Reply:
Added in https://github.com/fluent/fluent-bit-docs/pull/1542/commits/97c73c67311408ddaaf97a1e59e5524e199dab1b.